src/index.html

```diff
@@ -202,42 +203,42 @@ <h2>HOBBIES & INTERESTS</h2>
         </li>
       </ul>
 
-      <!-- Skills -->
-      <section>
-        <h2>SKILLS</h2>
-        <ul>
-          <li>
-            <div>Microsoft Excel</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Microsoft Word</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Adobe</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>Spanish</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>PowerPoint</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>French</div>
-            <div>Native</div>
-          </li>
-        </ul>
+      <section id="skills">
+        <div class="content">
+          <h2>SKILLS</h2>
+          <ul>
+            <li class="excel">
+              <div class="type">Microsoft Excel</div>
+              <div class="level">Advanced</div>
+            </li>
+            <li class="word">
+              <div class="type">Microsoft Word</div>
+              <div class="level">Advanced</div>
+            </li>
+            <li class="adobe">
+              <div class="type">Adobe</div>
+              <div class="level">Proficient</div>
+            </li>
+            <li class="spanish">
+              <div class="type">Spanish</div>
+              <div class="level">Advanced</div>
+            </li>
+            <li class="ppt">
+              <div class="type">PowerPoint</div>
+              <div class="level">Proficient</div>
+            </li>
+            <li class="french">
+              <div class="type">French</div>
+              <div class="level">Native</div>
+            </li>
+          </ul>
+        </div>
       </section>
 
       <div id="information">
         <section class="quick-id">
           <div class="content">
             <h2>QUICK ID</h2>
-            <hr />
```

Comment:
Replace `hr` with the after pseudo class of `h2`. Please refer to the related code: https://github.com/nzittraining/cv-assignment/pull/47/files#diff-0222076afe016b78b2300daa56069839R44-R51